packages/fast-usdc/README.md

```diff
@@ -59,43 +59,39 @@ sequenceDiagram
 
 # Status Manager
 
-### Pending Advance State Diagram
+### State Diagram
 
-*Transactions are qualified by the OCW and EventFeed before arriving to the Advancer.*
+*Transactions are qualified by the OCW and TransactionFeed before being
+delivered to the Advancer.*
 
-```mermaid
-stateDiagram-v2
-  [*] --> Observed: observe()
-  [*] --> Advancing: advancing()
-
-  Advancing --> Advanced: advanceOutcome(...true)
-  Advancing --> AdvanceFailed: advanceOutcome(...false)
-
-  Observed --> [*]: dequeueStatus()
-  Advanced --> [*]: dequeueStatus()
-  AdvanceFailed --> [*]: dequeueStatus()
-
-  note right of [*]
-    After dequeueStatus():
-    Transaction is removed 
-    from pendingTxs store.
-    Settler will .disburse()
-    or .forward()
-  end note
-```
+The transactionFeed receives attestations from Oracles, records their
+evidence, and when enough oracles agree, (if no risks are identified)
+it publishes the results for the advancer to act on.
+
+The Advancer subscribes (via `handleTransactionEvent`) to events published by
+the transactionFeed. When notified of an appropriate opportunity, it is
+responsible for advancing funds to fastUSDC payees.
 
-### Complete state diagram (starting from Transaction Feed into Advancer)
+The Settler is responsible for monitoring (via `receiveUpcall`) deposits to the
+settlementAccount. It either `disburse`s funds to the Pool (if funds were
+`advance`d to the payee), or `forwards` funds to the payee (if pool funds
+were not `advance`d).
 
 ```mermaid
 stateDiagram-v2
+  state Forwarding <<choice>>
+  state AdvancingChoice <<choice>>
+
   Observed --> AdvanceSkipped : Risks identified
   Observed --> Advancing : No risks, can advance
   Observed --> Forwarding : No risks, Mint deposited before advance
-  Forwarding --> Forwarded
-  Advancing --> Advanced
-  Advanced --> Disbursed
+
+  Forwarding --> Forwarded : settler.forward() succeeds
+  Advancing --> AcvancingChoice
+  AcvancingChoice --> Advanced : advancer's transferHandler detects success
+  Advanced --> Disbursed : settler.disburse()
   AdvanceSkipped --> Forwarding : Mint deposited
   AdvanceFailed --> Forwarding : Mint deposited
-  Advancing --> AdvanceFailed
-  Forwarding --> ForwardFailed
-```
+  AcvancingChoice --> AdvanceFailed : advancer's transferHandler detects failure
+  Forwarding --> ForwardFailed : settler.forward() fails
```

Comment:
Should we see `Forwarding`? Not seeing it in the rendered diagram for some reason.

<img width="396" alt="image" src="https://github.com/user-attachments/assets/7ed667bd-41b4-4590-8253-053b3474f0ce" />

Reply:
choice nodes render just as the diamond, not with their label